lib/event_manager_web/resolvers/attendances.ex

```diff
@@ -0,0 +1,34 @@
+defmodule EventManagerWeb.Resolvers.Attendances do
+  @moduledoc """
+  Resolvers for Attendance objects and related queries.
+  """
+
+  # alias Absinthe.Relay.Connection
+  alias EventManager.Attendances
+  # alias Phoenix.PubSub
```

Comment:
I would start the habit of emitting PubSub events every time we have a change in the application state (`created`, `updated`, `deleted` events or any other event that's signaling a change in the application). We never know when we could want to attach side effects to actions, so having the events already published will avoid us to search for the correct place to emit them.

@bernardini687 @oeN @apellizzn what do you think?

In this case, I would emit an `attendance:created` or `attendance:submitted` or whatever so that we can attach a listener to send an email to the user for confirmation, and one to send an email to the event creator for notification..

Reply:
Agree, I'll fix this.